opts/port_test.go

```diff
@@ -245,6 +245,38 @@ func TestPortOptInvalidComplexSyntax(t *testing.T) {
 	}
 }
 
+func TestPortOptInvalidSimpleSyntax(t *testing.T) {
+	testCases := []struct {
+		value         string
+		expectedError string
+	}{
+		{
+			value:         "9999999",
+			expectedError: "Invalid containerPort: 9999999",
```

Comment:
@AkihiroSuda yep, but this errors are from another package so I don't have "here" a way to change that 👼